.github/workflows/review.yaml

```diff
@@ -0,0 +1,52 @@
+
+name: PR Daily
+
+on:
+  schedule:
+    - cron: "10 1 * * *" # once a day
+
+  pull_request:
+    # test PRs targeting this branch code
+    branches: [ "master" ]
+
+  # allows to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+concurrency:
+  # Cancel ongoing tests in case of push to the same PR or staging branch,
+  # but let previous master commit tests complete.
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/master' }}
+
+env:
+  # empty except for pull_request events
+  PULL_REQUEST_NUMBER: ${{ github.event.pull_request.number }}
+
+  # Full clones of Squid repository branches (depth=19000+) waste resources,
+  # while excessively shallow clones break tests that check for past commits
+  # (e.g., to skip a particular test until a known bug is fixed) or generate
+  # diffs against them (e.g., for `git diff --check`). This arbitrary limit
+  # tries to balance the two concerns.
+  CHECKOUT_FETCH_DEPTH: 1001
```

Comment:
do we need to care for this any longer? In the end, it's github's resources, and they have plenty

Reply:
IMO the running time for the test is more what we care about. So avoiding a lot of data copy is likely to still be of benefit that way.

I am not sure what relation this value has to the "fetch-depth:" parameter on the job itself (one place where it was used). So left this matching the other files and set the job fetch-depth to the one commit we actually use.

Reply:
Feels like premature optimziation. We're talking about seconds at worst. I won't block on this, but I also don't think it should be included.

Reply:
Why not include?

Reply:
Okay, removed in c1d68a6